metadata/maelstrom.ttl

```diff
@@ -12,7 +12,7 @@
   dcterms:title "Maelstrom Aires d'information"@fr ;
   dc:description "Information classification developed by Maelstrom Research"@en ;
   dc:description "Classification des informations développée par Maelstrom Research"@fr ;
-  dcterms:license <https://creativecommons.org/licenses/by-nc-nd/4.0/> ;
+  dcterms:license <https://creativecommons.org/licenses/by/4.0/> ;
   dcterms:rights "You are free to:
```

Comment:
These `dcterms:rights` are for the old BY-NC-ND. They need to be updated.

Reply:
Fixed!